src/hooks/useVenueAccess.ts

```diff
@@ -1,26 +1,31 @@
-import { useEffect } from "react";
+import { useCallback, useEffect } from "react";
 import firebase from "firebase/app";
 
 import { AnyVenue } from "types/Firestore";
 
 import { WithId } from "utils/id";
-import { accessTokenKey } from "utils/localStorage";
-import { isTruthy, notEmpty } from "utils/types";
-import { checkAccess, getLocalStorageToken } from "functions/auth";
+import {
+  accessTokenKey,
+  checkAccess,
+  getLocalStorageToken,
+} from "utils/localStorage";
+import { isTruthy } from "utils/types";
 
-export const useVenueAccessToken = (
+export const useVenueAccess = (
   venue?: WithId<AnyVenue>,
   onDenyAccess?: () => void
 ) => {
-  useEffect(() => {
+  const denyAccess = useCallback(() => {
     if (!venue) return;
 
-    const denyAccess = () => {
-      localStorage.removeItem(accessTokenKey(venue.id));
-      onDenyAccess && onDenyAccess();
-    };
+    localStorage.removeItem(accessTokenKey(venue.id));
```

Comment:
Since we have all of the other `localStorage` helpers in `utils/localStorage`, it would be good for this to also be there. Maybe called `removeLocalStorageToken` or `clearLocalStorageToken`

Reply:
This is the only usage and usually I'm using the [rule of three](https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming)), (refactor if it's used 3 or more times). But I'm still going to change it

Reply:
Hadn't come across that rule specifically before.

I think my main 'concern' here was more around how we had that utils file with some of the related methods, yet we also do it directly outside of that file. If I came across that utils file, I would assume that that contains all of the localStorage related code, which it doesn't. So we should pick one or the other, and then apply that consistently (which seems to be using that file)

Thanks for making the change.